evm/LedgerSigner.js

```diff
@@ -1,5 +1,7 @@
 'use strict';
 
+require('ts-node/register'); /* enable node during migration */
```

Comment:
only scripts using `common/cli-utils.js` should be called via the cmd line, so the migration require can be omitted from other files?